src/Controller/ApiOpenStreetMapController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Controller;
+
+use App\Repository\ArtCardRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
+
+
+class ApiOpenStreetMapController extends AbstractController
+{
+
+    #[Route("/oneArt/{id}", name: 'api_home')]
```

Comment:
Nope, api_home is not a clear enough name for that route

Reply:
And as we are there, the URL too is not well named it should be "locateArt" or "getArtCoordinates"